homeassistant/components/transmission/strings.json

```diff
@@ -13,12 +13,11 @@
       }
     },
     "error": {
-      "name_exists": "Name already exists",
-      "wrong_credentials": "Wrong username or password",
-      "cannot_connect": "Unable to Connect to host"
+      "wrong_credentials": "[%key:common::config_flow::error::invalid_auth%]",
+      "cannot_connect": "[%key:common::config_flow::error::cannot_connect%]"
     },
     "abort": {
-      "already_configured": "Host is already configured."
```

Comment:
Maybe not renaming the key and use `[%key:common::config_flow::abort::already_configured_device%]`

Reply:
It was recommended to update the keys as well here:
https://github.com/home-assistant/core/pull/40590/commits/224e19d5bd05cad5326df6c690fd754afc7fe2fa#r495470098

Reply:
Also single_instance_allowed is better because we are not configuring device here.